### ISSUE TITLE
Bad french translation in 4.0.5 for inputTooShort

### ISSUE THREAD

**Issue by ragusa87** [select2/select2#5190]
Hello,

Since issue #4988 the french translation is broken for inputTooShort,
You can see an example here:
<https://jsbin.com/hinadixacu/edit?html,output>
![image](https://user-images.githubusercontent.com/1695207/34723748-ebfb9f44-f54b-11e7-9d81-e6ca6fa61b10.png)
The translation only displays an "s".
the code used is
```js
var message = 'Saisissez au moins ' + remainingChars + ' caractère' + (remainingChars>1)?'s':'';
```
instead of
```js
var message = 'Saisissez au moins ' + remainingChars + ' caractère' + (remainingChars>1?'s':'');
```

Be sure to fix it before 4.0.6.
Thanks

**Comment by alexweissman** [comment]
This has already been fixed in the 4.0.6 release candidate.  In the future you must check the [change log](https://github.com/select2/select2/blob/develop/CHANGELOG.md#translations) on the `develop` branch before opening a new issue.

**Comment by ragusa87** [comment]
Thanks, my bad. I checked master instead.
